Userland/Libraries/LibCodeComprehension/Cpp/Tests.cpp

```diff
@@ -55,7 +55,7 @@ class FileDB : public CodeComprehension::FileDB {
         if (project_root().has_value() && filename.starts_with(*project_root())) {
             target_filename = LexicalPath::relative_path(filename, *project_root());
         }
-        return m_map.get(target_filename);
+        return static_cast<Optional<ByteString>>(m_map.get(target_filename));
```

Comment:
Returning an Optional<T> from a function that internally calls something returning an Optional<T&> is somewhat common (as you know, given you touched all instances in this PR 🙂). This looks very verbose now. Maybe we could make that a bit nicer? E.g. by giving Optional<T&> a method returning an Optional<T> with a succinct name. `deref()` is taken /confusing though 😛 Maybe `copy()`?